buildSrc/src/integTest/groovy/org/elasticsearch/gradle/internal/test/rest/YamlRestCompatTestPluginFuncTest.groovy

```diff
@@ -220,7 +221,7 @@ class YamlRestCompatTestPluginFuncTest extends AbstractRestResourcesFuncTest {
         file("distribution/bwc/minor/checkoutDir/src/yamlRestTest/resources/rest-api-spec/test/test.yml" ) << """
         "one":
           - do:
-              get:
+              do.some.key_to_replace:
```

Comment:
can you remove the `do.` or change it to `do_`? it appears as a flatten `do.do` JSON structure, when it is only a descriptor

Reply:
Sure thing, 1a9b174